translations/de_CH/atrium-translations-de_CH-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionAnyAssertion.kt

```diff
@@ -12,5 +12,6 @@ enum class DescriptionAnyAssertion(override val value: String) : StringBasedTran
     IS_A("ist eine Instanz vom Typ"),
     IS_SAME("ist dieselbe Instanz wie"),
     IS_NOT_SAME("ist nicht dieselbe Instanz wie"),
-    IS_NONE_OF("ist nicht")
+    IS_NONE_OF("ist nicht"),
+    BECAUSE("weil")
```

Comment:
I think this should be ‘denn’, because ‘denn’ can be continued with the same grammatical structure as ‘because’. I fear with ‘weil’ we’ll get a lot of „weil: ? sind nicht in Dateinamen erlaubt“, which hurts!

Reply:
good with me, going to change it